lib/bns/domain/support_email.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Domain
+  ##
+  # The Domain::SupportEmail class provides a domain-specific representation of an SupporEmail object.
+  # It encapsulates information about a support email, including the subject, the sender, and the date.
+  #
+  class SupportEmail
```

Comment:
What would be the difference between a `SupportEmail` and just an `Email` based on the attributes? 🤔

Reply:
@FelipeGuzmanSierra Ok thanks for clarifying, let's use a more generic term for now, `Email`, then when new use cases appear we can evaluate if the domain object should be modified or new ones should be included